### ISSUE TITLE
Specifying the wrong conda-bld path

### ISSUE THREAD

**Issue by munechika-koyo** [conda-forge/conda-smithy#2168]
### Solution to issue cannot be found in the documentation.

- [x] I checked the documentation.

### Issue

When I use pixi for `conda_install_tool` in `conda-forge.yaml`, I encountered the following error at the `inspect_artifacts` command:
```bash
2024-11-24T16:51:47.3606890Z Adding in variants from internal_defaults
2024-11-24T16:51:47.3607700Z Adding in variants from argument_variants
2024-11-24T16:51:47.3627730Z Traceback (most recent call last):
2024-11-24T16:51:47.3628490Z   File "/Users/runner/work/1/s/.pixi/envs/default/bin/validate_recipe_outputs", line 11, in <module>
2024-11-24T16:51:47.3629250Z     sys.exit(main())
2024-11-24T16:51:47.3629690Z              ^^^^^^
2024-11-24T16:51:47.3630690Z   File "/Users/runner/work/1/s/.pixi/envs/default/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
2024-11-24T16:51:47.3632160Z     return self.main(*args, **kwargs)
2024-11-24T16:51:47.3632940Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-24T16:51:47.3633910Z   File "/Users/runner/work/1/s/.pixi/envs/default/lib/python3.12/site-packages/click/core.py", line 1078, in main
2024-11-24T16:51:47.3635500Z     rv = self.invoke(ctx)
2024-11-24T16:51:47.3636030Z          ^^^^^^^^^^^^^^^^
2024-11-24T16:51:47.3637200Z   File "/Users/runner/work/1/s/.pixi/envs/default/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
2024-11-24T16:51:47.3639770Z     return ctx.invoke(self.callback, **ctx.params)
2024-11-24T16:51:47.3640350Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-24T16:51:47.3641480Z   File "/Users/runner/work/1/s/.pixi/envs/default/lib/python3.12/site-packages/click/core.py", line 783, in invoke
2024-11-24T16:51:47.3642690Z     return __callback(*args, **kwargs)
2024-11-24T16:51:47.3643210Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-24T16:51:47.3644150Z   File "/Users/runner/work/1/s/.pixi/envs/default/lib/python3.12/site-packages/conda_forge_ci_setup/feedstock_outputs.py", line 158, in main
2024-11-24T16:51:47.3654540Z     distributions = built_distributions_from_recipe_variant(recipe_dir=recipe_dir, variant=variant)
2024-11-24T16:51:47.3655280Z                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-24T16:51:47.3656470Z   File "/Users/runner/work/1/s/.pixi/envs/default/lib/python3.12/site-packages/conda_forge_ci_setup/utils.py", line 126, in built_distributions_from_recipe_variant
2024-11-24T16:51:47.3657290Z     for dist in built_distributions(subdirs=allowed_subdirs)
2024-11-24T16:51:47.3657860Z                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-24T16:51:47.3659000Z   File "/Users/runner/work/1/s/.pixi/envs/default/lib/python3.12/site-packages/conda_forge_ci_setup/utils.py", line 109, in built_distributions
2024-11-24T16:51:47.3660250Z     for path in os.listdir(os.path.join(conda_build.config.croot, subdir)):
2024-11-24T16:51:47.3660760Z                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-11-24T16:51:47.3661640Z FileNotFoundError: [Errno 2] No such file or directory: '/Users/runner/work/1/s/.pixi/envs/default/conda-bld/osx-arm64'

```
I guess `conda_forge_ci_setup` specified the wrong path to the `conda-bld` directory.
Temporally, Setting `CONDA_BLD_PATH` env before the `inspect_artifacts` command avoids the above in this PR:https://github.com/conda-forge/raysect-feedstock/pull/4

### Installed packages

```shell
pixi list
Package                     Version       Build               Size       Kind   Source
anaconda-client             1.12.3        pyhd8ed1ab_1        70.8 KiB   conda  anaconda-client-1.12.3-pyhd8ed1ab_1.conda
archspec                    0.2.3         pyhd8ed1ab_0        47.6 KiB   conda  archspec-0.2.3-pyhd8ed1ab_0.conda
attrs                       24.2.0        pyh71513ae_0        54.7 KiB   conda  attrs-24.2.0-pyh71513ae_0.conda
beautifulsoup4              4.12.3        pyha770c72_0        115.4 KiB  conda  beautifulsoup4-4.12.3-pyha770c72_0.conda
boltons                     24.0.0        pyhd8ed1ab_0        290.9 KiB  conda  boltons-24.0.0-pyhd8ed1ab_0.conda
brotli-python               1.1.0         py312h5861a67_2     354.7 KiB  conda  brotli-python-1.1.0-py312h5861a67_2.conda
bzip2                       1.0.8         hfdf4475_7          131 KiB    conda  bzip2-1.0.8-hfdf4475_7.conda
c-ares                      1.34.3        hf13058a_1          180.1 KiB  conda  c-ares-1.34.3-hf13058a_1.conda
ca-certificates             2024.8.30     h8857fd0_0          154.9 KiB  conda  ca-certificates-2024.8.30-h8857fd0_0.conda
cctools                     1010.6        h92db3b5_1          21 KiB     conda  cctools-1010.6-h92db3b5_1.conda
cctools_osx-64              1010.6        hd0ff2cf_1          1 MiB      conda  cctools_osx-64-1010.6-hd0ff2cf_1.conda
certifi                     2024.8.30     pyhd8ed1ab_0        159.9 KiB  conda  certifi-2024.8.30-pyhd8ed1ab_0.conda
cffi                        1.17.1        py312hf857d28_0     275.8 KiB  conda  cffi-1.17.1-py312hf857d28_0.conda
chardet                     5.2.0         py312hb401068_2     255.4 KiB  conda  chardet-5.2.0-py312hb401068_2.conda
charset-normalizer          3.4.0         pyhd8ed1ab_0        46.2 KiB   conda  charset-normalizer-3.4.0-pyhd8ed1ab_0.conda
click                       8.1.7         unix_pyh707e725_0   82.5 KiB   conda  click-8.1.7-unix_pyh707e725_0.conda
colorama                    0.4.6         pyhd8ed1ab_0        24.6 KiB   conda  colorama-0.4.6-pyhd8ed1ab_0.tar.bz2
conda                       24.9.2        py312hb401068_0     1.1 MiB    conda  conda-24.9.2-py312hb401068_0.conda
conda-build                 24.9.0        py312hb401068_1     738.1 KiB  conda  conda-build-24.9.0-py312hb401068_1.conda
conda-env                   2.6.0         1                   2.3 KiB    conda  conda-env-2.6.0-1.tar.bz2
conda-forge-ci-setup        4.12.2        py312hd35cd3b_100   75 KiB     conda  conda-forge-ci-setup-4.12.2-py312hd35cd3b_100.conda
conda-forge-metadata        0.11.0        pyhd8ed1ab_0        17.6 KiB   conda  conda-forge-metadata-0.11.0-pyhd8ed1ab_0.conda
conda-index                 0.5.0         pyhd8ed1ab_0        179.8 KiB  conda  conda-index-0.5.0-pyhd8ed1ab_0.conda
conda-libmamba-solver       24.9.0        pyhd8ed1ab_0        40.6 KiB   conda  conda-libmamba-solver-24.9.0-pyhd8ed1ab_0.conda
conda-oci-mirror            0.2.1         pyhd8ed1ab_0        31.7 KiB   conda  conda-oci-mirror-0.2.1-pyhd8ed1ab_0.conda
conda-package-handling      2.4.0         pyh7900ff3_0        251.7 KiB  conda  conda-package-handling-2.4.0-pyh7900ff3_0.conda
conda-package-streaming     0.11.0        pyhd8ed1ab_0        20.1 KiB   conda  conda-package-streaming-0.11.0-pyhd8ed1ab_0.conda
defusedxml                  0.7.1         pyhd8ed1ab_0        23.5 KiB   conda  defusedxml-0.7.1-pyhd8ed1ab_0.tar.bz2
deprecated                  1.2.15        pyhff2d567_0        13.8 KiB   conda  deprecated-1.2.15-pyhff2d567_0.conda
distro                      1.9.0         pyhd8ed1ab_0        41.1 KiB   conda  distro-1.9.0-pyhd8ed1ab_0.conda
filelock                    3.16.1        pyhd8ed1ab_0        17 KiB     conda  filelock-3.16.1-pyhd8ed1ab_0.conda
fmt                         11.0.2        h3c5361c_0          180.1 KiB  conda  fmt-11.0.2-h3c5361c_0.conda
frozendict                  2.4.6         py312h3d0f464_0     30 KiB     conda  frozendict-2.4.6-py312h3d0f464_0.conda
git                         2.47.0        pl5321ha198fdc_0    10 MiB     conda  git-2.47.0-pl5321ha198fdc_0.conda
h2                          4.1.0         pyhd8ed1ab_0        45.7 KiB   conda  h2-4.1.0-pyhd8ed1ab_0.tar.bz2
hpack                       4.0.0         pyh9f0ad1d_0        24.7 KiB   conda  hpack-4.0.0-pyh9f0ad1d_0.tar.bz2
hyperframe                  6.0.1         pyhd8ed1ab_0        14.3 KiB   conda  hyperframe-6.0.1-pyhd8ed1ab_0.tar.bz2
icu                         75.1          h120a0e1_0          11.2 MiB   conda  icu-75.1-h120a0e1_0.conda
idna                        3.10          pyhd8ed1ab_0        48.7 KiB   conda  idna-3.10-pyhd8ed1ab_0.conda
importlib_resources         6.4.5         pyhd8ed1ab_0        32 KiB     conda  importlib_resources-6.4.5-pyhd8ed1ab_0.conda
jinja2                      3.1.4         pyhd8ed1ab_0        109 KiB    conda  jinja2-3.1.4-pyhd8ed1ab_0.conda
joblib                      1.4.2         pyhd8ed1ab_0        214.6 KiB  conda  joblib-1.4.2-pyhd8ed1ab_0.conda
jq                          1.7.1         h10d778d_0          296.8 KiB  conda  jq-1.7.1-h10d778d_0.conda
jsonpatch                   1.33          pyhd8ed1ab_0        17 KiB     conda  jsonpatch-1.33-pyhd8ed1ab_0.conda
jsonpointer                 3.0.0         py312hb401068_1     17.1 KiB   conda  jsonpointer-3.0.0-py312hb401068_1.conda
jsonschema                  4.23.0        pyhd8ed1ab_0        72.6 KiB   conda  jsonschema-4.23.0-pyhd8ed1ab_0.conda
jsonschema-specifications   2024.10.1     pyhd8ed1ab_0        15.8 KiB   conda  jsonschema-specifications-2024.10.1-pyhd8ed1ab_0.conda
jupyter_core                5.7.2         pyh31011fe_1        56.3 KiB   conda  jupyter_core-5.7.2-pyh31011fe_1.conda
krb5                        1.21.3        h37d8d59_0          1.1 MiB    conda  krb5-1.21.3-h37d8d59_0.conda
ld64                        951.9         hc3792c1_1          18.3 KiB   conda  ld64-951.9-hc3792c1_1.conda
ld64_osx-64                 951.9         h3c7722d_1          1 MiB      conda  ld64_osx-64-951.9-h3c7722d_1.conda
libarchive                  3.7.4         h20e244c_0          725.3 KiB  conda  libarchive-3.7.4-h20e244c_0.conda
libcurl                     8.10.1        h58e7537_0          393.2 KiB  conda  libcurl-8.10.1-h58e7537_0.conda
libcxx                      19.1.4        hf95d169_0          516.6 KiB  conda  libcxx-19.1.4-hf95d169_0.conda
libedit                     3.1.20191231  h0678c8f_2          102.9 KiB  conda  libedit-3.1.20191231-h0678c8f_2.tar.bz2
libev                       4.33          h10d778d_2          104.2 KiB  conda  libev-4.33-h10d778d_2.conda
libexpat                    2.6.4         h240833e_0          69.1 KiB   conda  libexpat-2.6.4-h240833e_0.conda
libffi                      3.4.2         h0d85af4_5          50.1 KiB   conda  libffi-3.4.2-h0d85af4_5.tar.bz2
libiconv                    1.17          hd75f5a5_2          650.9 KiB  conda  libiconv-1.17-hd75f5a5_2.conda
libintl                     0.22.5        hdfe23c8_3          86 KiB     conda  libintl-0.22.5-hdfe23c8_3.conda
liblief                     0.14.1        hac325c4_2          1.6 MiB    conda  liblief-0.14.1-hac325c4_2.conda
libllvm19                   19.1.4        hc29ff6c_0          27.5 MiB   conda  libllvm19-19.1.4-hc29ff6c_0.conda
libmamba                    1.5.11        hd41e4cc_0          1.2 MiB    conda  libmamba-1.5.11-hd41e4cc_0.conda
libmambapy                  1.5.11        py312h0252a60_0     267.5 KiB  conda  libmambapy-1.5.11-py312h0252a60_0.conda
libnghttp2                  1.64.0        hc7306c3_0          592.4 KiB  conda  libnghttp2-1.64.0-hc7306c3_0.conda
libsolv                     0.7.30        h69d5d9b_0          405.9 KiB  conda  libsolv-0.7.30-h69d5d9b_0.conda
libsqlite                   3.47.0        h2f8c449_1          893.8 KiB  conda  libsqlite-3.47.0-h2f8c449_1.conda
libssh2                     1.11.1        h3dc7d44_0          277.2 KiB  conda  libssh2-1.11.1-h3dc7d44_0.conda
libxml2                     2.13.5        h495214b_0          594.7 KiB  conda  libxml2-2.13.5-h495214b_0.conda
libzlib                     1.3.1         hd23fc13_2          55.8 KiB   conda  libzlib-1.3.1-hd23fc13_2.conda
llvm-tools                  19.1.4        h3fe3016_0          85 KiB     conda  llvm-tools-19.1.4-h3fe3016_0.conda
llvm-tools-19               19.1.4        he90a8e3_0          16.6 MiB   conda  llvm-tools-19-19.1.4-he90a8e3_0.conda
lz4-c                       1.9.4         hf0c8a7f_0          152.7 KiB  conda  lz4-c-1.9.4-hf0c8a7f_0.conda
lzo                         2.10          h10d778d_1001       143 KiB    conda  lzo-2.10-h10d778d_1001.conda
mamba                       1.5.11        py312ha12221d_0     63.7 KiB   conda  mamba-1.5.11-py312ha12221d_0.conda
markupsafe                  3.0.2         py312hbe3f5e4_0     23.3 KiB   conda  markupsafe-3.0.2-py312hbe3f5e4_0.conda
menuinst                    2.2.0         py312hb401068_0     162.8 KiB  conda  menuinst-2.2.0-py312hb401068_0.conda
more-itertools              10.5.0        pyhd8ed1ab_0        56 KiB     conda  more-itertools-10.5.0-pyhd8ed1ab_0.conda
nbformat                    5.10.4        pyhd8ed1ab_0        98.9 KiB   conda  nbformat-5.10.4-pyhd8ed1ab_0.conda
ncurses                     6.5           hf036a51_1          802.8 KiB  conda  ncurses-6.5-hf036a51_1.conda
oniguruma                   6.9.9         h10d778d_0          219.2 KiB  conda  oniguruma-6.9.9-h10d778d_0.conda
openssl                     3.4.0         hd471939_0          2.5 MiB    conda  openssl-3.4.0-hd471939_0.conda
oras-py                     0.1.14        pyhd8ed1ab_0        33.6 KiB   conda  oras-py-0.1.14-pyhd8ed1ab_0.conda
packaging                   24.2          pyhff2d567_1        59 KiB     conda  packaging-24.2-pyhff2d567_1.conda
patch                       2.7.6         hbcf498f_1002       132.8 KiB  conda  patch-2.7.6-hbcf498f_1002.tar.bz2
pcre2                       10.44         h7634a1b_2          834.3 KiB  conda  pcre2-10.44-h7634a1b_2.conda
perl                        5.32.1        7_h10d778d_perl5    11.8 MiB   conda  perl-5.32.1-7_h10d778d_perl5.conda
pip                         24.3.1        pyh8b19718_0        1.2 MiB    conda  pip-24.3.1-pyh8b19718_0.conda
pkginfo                     1.11.2        pyhd8ed1ab_0        29 KiB     conda  pkginfo-1.11.2-pyhd8ed1ab_0.conda
pkgutil-resolve-name        1.3.10        pyhd8ed1ab_1        10.5 KiB   conda  pkgutil-resolve-name-1.3.10-pyhd8ed1ab_1.conda
platformdirs                4.3.6         pyhd8ed1ab_0        20.1 KiB   conda  platformdirs-4.3.6-pyhd8ed1ab_0.conda
pluggy                      1.5.0         pyhd8ed1ab_0        23.3 KiB   conda  pluggy-1.5.0-pyhd8ed1ab_0.conda
psutil                      6.1.0         py312h3d0f464_0     481.9 KiB  conda  psutil-6.1.0-py312h3d0f464_0.conda
py-lief                     0.14.1        py312h5861a67_2     526.4 KiB  conda  py-lief-0.14.1-py312h5861a67_2.conda
pybind11-abi                4             hd8ed1ab_3          9.7 KiB    conda  pybind11-abi-4-hd8ed1ab_3.tar.bz2
pycosat                     0.6.6         py312h104f124_0     87.1 KiB   conda  pycosat-0.6.6-py312h104f124_0.conda
pycparser                   2.22          pyhd8ed1ab_0        102.6 KiB  conda  pycparser-2.22-pyhd8ed1ab_0.conda
pysocks                     1.7.1         pyha2e5f31_6        18.5 KiB   conda  pysocks-1.7.1-pyha2e5f31_6.tar.bz2
python                      3.12.7        h8f8b54e_0_cpython  13.1 MiB   conda  python-3.12.7-h8f8b54e_0_cpython.conda
python-dateutil             2.9.0.post0   pyhff2d567_0        216.7 KiB  conda  python-dateutil-2.9.0.post0-pyhff2d567_0.conda
python-fastjsonschema       2.20.0        pyhd8ed1ab_0        220.9 KiB  conda  python-fastjsonschema-2.20.0-pyhd8ed1ab_0.conda
python-libarchive-c         5.1           py312hb401068_1     68.4 KiB   conda  python-libarchive-c-5.1-py312hb401068_1.conda
python_abi                  3.12          5_cp312             6.2 KiB    conda  python_abi-3.12-5_cp312.conda
pytz                        2024.2        pyhd8ed1ab_0        182.6 KiB  conda  pytz-2024.2-pyhd8ed1ab_0.conda
pyyaml                      6.0.2         py312hb553811_1     184.9 KiB  conda  pyyaml-6.0.2-py312hb553811_1.conda
rattler-build               0.31.1        h0f13631_0          9 MiB      conda  rattler-build-0.31.1-h0f13631_0.conda
rattler-build-conda-compat  1.2.2         pyhd8ed1ab_0        33.7 KiB   conda  rattler-build-conda-compat-1.2.2-pyhd8ed1ab_0.conda
readline                    8.2           h9e318b2_1          249.9 KiB  conda  readline-8.2-h9e318b2_1.conda
referencing                 0.35.1        pyhd8ed1ab_0        41.2 KiB   conda  referencing-0.35.1-pyhd8ed1ab_0.conda
reproc                      14.2.5.post0  h6e16a3a_0          31 KiB     conda  reproc-14.2.5.post0-h6e16a3a_0.conda
reproc-cpp                  14.2.5.post0  h240833e_0          23.8 KiB   conda  reproc-cpp-14.2.5.post0-h240833e_0.conda
requests                    2.32.3        pyhd8ed1ab_0        57.4 KiB   conda  requests-2.32.3-pyhd8ed1ab_0.conda
requests-toolbelt           1.0.0         pyhd8ed1ab_0        42.9 KiB   conda  requests-toolbelt-1.0.0-pyhd8ed1ab_0.conda
ripgrep                     14.1.1        h926acf8_0          1.5 MiB    conda  ripgrep-14.1.1-h926acf8_0.conda
rpds-py                     0.21.0        py312h0d0de52_0     296.7 KiB  conda  rpds-py-0.21.0-py312h0d0de52_0.conda
ruamel.yaml                 0.18.6        py312h3d0f464_1     260.7 KiB  conda  ruamel.yaml-0.18.6-py312h3d0f464_1.conda
ruamel.yaml.clib            0.2.8         py312h3d0f464_1     119.5 KiB  conda  ruamel.yaml.clib-0.2.8-py312h3d0f464_1.conda
ruamel_yaml                 0.15.80       py312h104f124_1009  296.9 KiB  conda  ruamel_yaml-0.15.80-py312h104f124_1009.conda
setuptools                  75.6.0        pyhff2d567_0        756.2 KiB  conda  setuptools-75.6.0-pyhff2d567_0.conda
shyaml                      0.6.2         pyhd3deb0d_0        21 KiB     conda  shyaml-0.6.2-pyhd3deb0d_0.tar.bz2
sigtool                     0.1.3         h88f4db0_0          208.8 KiB  conda  sigtool-0.1.3-h88f4db0_0.tar.bz2
six                         1.16.0        pyh6c4a22f_0        13.9 KiB   conda  six-1.16.0-pyh6c4a22f_0.tar.bz2
soupsieve                   2.5           pyhd8ed1ab_1        35.9 KiB   conda  soupsieve-2.5-pyhd8ed1ab_1.conda
tapi                        1300.6.5      h390ca13_0          216.1 KiB  conda  tapi-1300.6.5-h390ca13_0.conda
tk                          8.6.13        h1abcd95_1          3.1 MiB    conda  tk-8.6.13-h1abcd95_1.conda
tomli                       2.1.0         pyhff2d567_0        18.3 KiB   conda  tomli-2.1.0-pyhff2d567_0.conda
tqdm                        4.67.0        pyhd8ed1ab_0        87.3 KiB   conda  tqdm-4.67.0-pyhd8ed1ab_0.conda
traitlets                   5.14.3        pyhd8ed1ab_0        107.6 KiB  conda  traitlets-5.14.3-pyhd8ed1ab_0.conda
truststore                  0.10.0        pyhd8ed1ab_0        21.3 KiB   conda  truststore-0.10.0-pyhd8ed1ab_0.conda
typing-extensions           4.12.2        hd8ed1ab_0          9.9 KiB    conda  typing-extensions-4.12.2-hd8ed1ab_0.conda
typing_extensions           4.12.2        pyha770c72_0        39 KiB     conda  typing_extensions-4.12.2-pyha770c72_0.conda
tzdata                      2024b         hc8b5060_0          119.5 KiB  conda  tzdata-2024b-hc8b5060_0.conda
urllib3                     2.2.3         pyhd8ed1ab_0        95.8 KiB   conda  urllib3-2.2.3-pyhd8ed1ab_0.conda
wheel                       0.45.1        pyhd8ed1ab_0        61.5 KiB   conda  wheel-0.45.1-pyhd8ed1ab_0.conda
wrapt                       1.16.0        py312hb553811_1     56.9 KiB   conda  wrapt-1.16.0-py312hb553811_1.conda
xz                          5.2.6         h775f41a_0          232.5 KiB  conda  xz-5.2.6-h775f41a_0.tar.bz2
yaml                        0.2.5         h0d85af4_2          82.3 KiB   conda  yaml-0.2.5-h0d85af4_2.tar.bz2
yaml-cpp                    0.8.0         he965462_0          127.3 KiB  conda  yaml-cpp-0.8.0-he965462_0.conda
zipp                        3.21.0        pyhd8ed1ab_0        21.2 KiB   conda  zipp-3.21.0-pyhd8ed1ab_0.conda
zstandard                   0.23.0        py312h7122b0e_1     401.2 KiB  conda  zstandard-0.23.0-py312h7122b0e_1.conda
zstd                        1.5.6         h915ae27_0          487.2 KiB  conda  zstd-1.5.6-h915ae27_0.conda
```

### Environment info

```shell
active environment : base
   active env location : /Users/runner/work/1/s/.pixi/envs/default
      user config file : /Users/runner/.condarc
populated config files : /Users/runner/work/1/s/.pixi/envs/default/.condarc
         conda version : 24.9.2
   conda-build version : 24.9.0
        python version : 3.12.7.final.0
                solver : libmamba (default)
      virtual packages : __archspec=1=skylake
                         __conda=24.9.2=0
                         __osx=13.7.1=0
                         __unix=0=0
      base environment : /Users/runner/work/1/s/.pixi/envs/default  (writable)
     conda av data dir : /Users/runner/work/1/s/.pixi/envs/default/etc/conda
 conda av metadata url : None
          channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                         https://conda.anaconda.org/conda-forge/noarch
         package cache : /Users/runner/work/1/s/.pixi/envs/default/pkgs
                         /Users/runner/.conda/pkgs
      envs directories : /Users/runner/work/1/s/.pixi/envs/default/envs
                         /Users/runner/.conda/envs
              platform : osx-64
            user-agent : conda/24.9.2 requests/2.32.3 CPython/3.12.7 Darwin/22.6.0 OSX/13.7.1 solver/libmamba conda-libmamba-solver/24.9.0 libmambapy/1.5.11
               UID:GID : 501:20
            netrc file : None
          offline mode : False
```

**Comment by hadim** [comment]
It seems similar to https://github.com/conda-forge/conda-smithy/issues/2145

**Comment by jaimergp** [comment]
Closed by #2148
